src/panel.js

```diff
@@ -256,7 +325,7 @@ document.addEventListener("DOMContentLoaded", async () => {
 
   const onboarding = (currentPanel.includes("onboarding"));
   if (!onboarding) {
-    return buildPanel(currentPanel);
+    return await buildPanel(currentPanel);
```

Comment:
nit: [don't really need to `return await`](https://eslint.org/docs/rules/no-return-await)